examples/angularjs/test/.gitignore

```diff
@@ -0,0 +1,15 @@
+node_modules/.bin/
+node_modules/karma/
+node_modules/karma-chrome-launcher/
+node_modules/karma-firefox-launcher/
+node_modules/karma-ie-launcher/
+node_modules/karma-jasmine/
+node_modules/karma-phantomjs-launcher/
+node_modules/jasmine-jquery/
+node_modules/jasmine-flight/
+node_modules/karma-safari-launcher/
+node_modules/karma-requirejs/
+node_modules/karma-coffee-preprocessor/
+node_modules/karma-html2js-preprocessor/
+node_modules/karma-script-launcher/
+node_modules/requirejs/
```

Comment:
You can just ignore the whole `test/node_modules` folder in the other .gitignore. We only care about files needed for running the app.